docs/UserGuide.adoc

```diff
@@ -243,6 +243,30 @@ transaction id of the parcels and the address of the customers, etc.
 * Import the specific csv with the `FILE_NAME` in *data* folder to the application.
 ====
 
+[IMPORTANT]
+====
+* The data in the `csv file` should be written in the following format:
+
+[%header, format=csv]
+.Data format in csv file
+|===
+ot/ORDER_TYPE, tid/TRANSACTION_ID, n/NAME, a/ADDRESS, p/PHONE_NUMBER, dts/DELIVERY_DATE_&_TIME (rts/RETURN_DATE_&_TIME), w/WAREHOUSE_LOCATION, pm/PAYMENT_METHOD, [c/COMMENTS_BY_CUSTOMER], [type/TYPE_OF_ITEM]
+|===
+
+
+* Examples:
+** ot/order, tid/1023456789, n/Amos Cheong, a/Blk 572 Hougang st 51 #11-37 S530572, p/9001 0019, dts/2020-03-10 1650, w/Marsiling, pm/Nets, c/Leave it at the riser, type/glass
+** ot/return, tid/b1230512, n/Aaron Teo, a/256 Alpha Road #03-22 S123567, p/91230456, rts/12-12-2020 1400, w/Jurong Warehouse, pm/visa, c/Leave it at the lobby, type/metal
```

Comment:
This whole chunk looks very big, I suggest, remove the table and use the commas as an example instead since csv files are all separated by commas. 
Also, please include another example for return as return don't have payment methods

Reply:
Resolved. Added another table for the data format for return.